web/client/components/security/modals/LoginModal.jsx

```diff
@@ -99,6 +99,13 @@ class LoginModal extends React.Component {
         </Modal>);
     }
 
+    handleOnHide = (event) => {
```

Comment:
please add some more comments here
- onHide is called two times from Modal of bootstrap, one has the event one has not, specify this.

also onClose can share this method